### ISSUE TITLE
Hangs on Login

### ISSUE THREAD

**Issue by lupin4** [aescripts/aescripts-updater-public#8]
AEScripts app never actually logs in, just hangs on login

**Comment by Maxweel** [comment]
This is an issue we're having trouble understanding, you're not the only
one with it.
First, are you on Mac or Windows, then do you have lots of products in your
account?

Thanks,

Kevin

2017-11-06 23:59 GMT+08:00 lupin4 <notifications@github.com>:

**Comment by lupin4** [comment]
Hi Kevin,

Thanks yes, I figured, its ok.  I am on Mac and I have almost 200 products.

**Comment by Maxweel** [comment]
Right, so you're like the others who couldn't get their account to update.
We really need people to help us debug that, if you'd be available please
open a support ticket so that I can contact you directly and send you debug
versions of the app: https://aescripts.com/contact/

Thanks,

Kevin

2017-11-07 10:54 GMT+08:00 lupin4 <notifications@github.com>:
